docs/source/tutorials & examples/index.rst

```diff
@@ -0,0 +1,13 @@
+.. _tutorials & examples:
+
+Tutorials & Examples
```

Comment:
I wonder if we could more broadly group the examples into a few categories, where each category corresponds to a specific user journey (a goal the end-user is trying to accomplish)

Below, you can find a number of tutorials for various MLflow use cases:

* Reproducibly run & share ML code
  * [Using docker environments](https://github.com/mlflow/mlflow/tree/master/examples/docker)
  * [Using conda environments](https://github.com/mlflow/mlflow/tree/master/examples/sklearn_elasticnet_wine)

* Instrument ML training code with MLflow
  * TensorFlow (link the sub-bullets)
  * Keras
  ...
  * Prophet
  * Gluon
  * R
 
* [Orchestrate Multistep Workflows](https://github.com/mlflow/mlflow/tree/master/examples/multistep_workflow)
* [Train, Serve and Score an Image-Classification Model](https://github.com/mlflow/mlflow/tree/master/examples/flower_classifier)

Reply:
Yeah that looks good!